library/control/src/modules/ProductFeatures.rb

```diff
@@ -241,6 +242,27 @@ def GetBooleanFeature(section, feature)
       Ops.is_string?(value) && Builtins.tolower(Convert.to_string(value)) == "yes"
     end
 
+    # Get value of a boolean feature with a fallback value.
+    #
+    # @note This is a stable API function
+    # @param [String] section string section of the feature
+    # @param [String] feature feature name
+    # @param [Boolean] fallback
+    #
+    # @return [Boolean] the feature value or fallback if not specified
+    def GetBooleanFeatureWithFallback(section, feature, fallback)
+      value = GetFeature(section, feature)
+      return fallback if value.nil?
+      return value if Ops.is_boolean?(value)
+
+      if value.respond_to?(:downcase)
+        return true  if ["yes", "true"].include?(value.downcase)
```

Comment:
Could this break in strange locales?

Reply:
No, those are values from the XML file, and this is used in other places in the same code as well.